Cargo.toml

```diff
@@ -41,7 +41,7 @@ routing = { path = "./routing" }
 
 # External
 afpacket = { version = "0.2.3", default-features = false, features = [] }
-ahash = { version = "0.8.11", default-features = false, features = [] }
+ahash = { git = "https://github.com/githedgehog/aHash", branch = "pr/daniel-noland/bump-zero-copy", default-features = false, features = ["no-rng"] }
```

Comment:
Why do we need to bump aHash to get a new version of zerocopy?

Reply:
The zerocopy bump is related to the cargo-deny requirement that we don't have duplicate versions of dependencies linked into our production artifacts.
Some other crates picked the new version of zerocopy but ahash hadn't updated.  This bounced our build